HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,163 @@
+2. Author(s) MAY discuss ideas in various Holochain community forums.
+3. Author(s) MAY get early feedback from core developers.
+4. This stage is formally complete when a pull request is made against the HOW repository.  The pull request must be made using the "Create HOW draft" template. Additionally, when submitting the pull request, Author(s) MUST check "allow edits and access to secrets by maintainers".
+
+#### Refine
+This is the review stage, when Author(s) are ready for feedback from the core developers and the community at large. The status for this stage are: `REVIEW`, `LAST CALL`.  The steps in this stage are:
+
+1. A member of the core team or requested working group will do an initial evaluation of the PR.  If the a team member of working group member is willing to take on stewardship of this HOW they will update the header status to `REVIEW` and will add the section(s) specified in `/social/process/how/_requirements.md#Process Template Section: REVIEW`.  If no requested steward is willing take on the HOW it MUST be updated to status `REJECTED` and the section(s) specified in `/social/process/how/_requirements.md#Process Template Section: REJECTED` added.
+2. Community members make comments and suggested changes on the pull request.
+3. Author(s) address feedback, making changes as desired, and keeping all commits in the git history without squashing or force-pushing.
+4. When most concerns appear to be addressed, the steward makes a call and changes the status to `LAST CALL` at that point the community and Author(s) have 14 days to raise and address any final concerns.
+5. Author(s) MAY chose to set the status to `WITHDRAWN` at any time during the Define and Refine stages, at which point the process terminates.
+
+#### Align
+This is the concluding stage where either the core team or the working group decides on the HOW.   The statuses for this stage are: `REJECTED`, `LIVE`, `DEFERRED`.  The steps in this stage are:
+
+1. What ever acceptance process is use by core team or working group is used (voting, consensus, coin-toss, etc) is used.
+2. Given the outcome of the process the status of the HOW is updated to `REJECTED`, `DEFERRED` or `LIVE` as appropriate along with addition of sections as per `/social/process/how/_requirements.md`
+3. In the case of a HOW being accepted, the PR is merged into the repository, keeping the original git history without squashing.
+
+```mermaid
+stateDiagram
+   direction LR
+
+    state Define {
+        DRAFT
+        DRAFT --> WITHDRAWN
+        DRAFT --> REVIEW
+        WITHDRAWN
+    }
+
+    state Refine {
+        REVIEW
+        REVIEW --> LAST_CALL
+        REVIEW --> WITHDRAWN
+        LAST_CALL
+        LAST_CALL --> REJECTED
+        LAST_CALL --> LIVE
+        LAST_CALL --> DEFERRED
+        LAST_CALL --> WITHDRAWN
+    }
+
+    state Align {
+        REJECTED
+        ALIVE
```

Comment:
Does `ALIVE` include actual implementation for holochain core standards? For app-level standards I guess `ALIVE` is if we agreed on it but for holochain core standards it's not really alive until it's actually implemented in code. But implementation is still considered as part of the `ALIGN` stage maybe? And what about different versions of holochain? A core standard may be implemented in verison 2.0 but not in version 1.0...

Reply:
Hmm that's a good catch. We should track that a core proposal has actually been implemented and at what holochain version.

For now though, since we're starting with HOW only including app-level tech standards, I'd be fine with tabling it.